austrakka/components/sequence/sync/constant.py

```diff
@@ -0,0 +1,54 @@
+# Sync state key constants
+MANIFEST_KEY = 'manifest'
+INTERMEDIATE_MANIFEST_FILE_KEY = 'intermediate_manifest_file'
+GROUP_NAME_KEY = 'group_name'
+SEQ_TYPE_KEY = 'seq_type'
+OUTPUT_DIR_KEY = 'output_dir'
+SYNC_STATE_FILE_KEY = 'sync_state_file'
+HASH_CHECK_KEY = 'hash_check'
+OBSOLETE_OBJECTS_FILE_KEY = 'obsolete_objects_file'
+CURRENT_STATE_KEY = 'current_state'
+CURRENT_ACTION_KEY = 'current_action'
+TRASH_DIR_KEY = 'trash_dir'
+
+# File extensions
```

Comment:
Probably need to purge `.fa` as well as `.fasta`, and `.fq` as well as `fastq`

Reply:
Although if these extensions are ONLY used for purging (?) and if I add a more paranoid regex instead of an extension check, this comment will be obsolete - list of extensions won't matter?